src/app/cluster/details/cluster/template.html

```diff
@@ -446,3 +455,8 @@
                color="accent"
                class="km-spinner"></mat-spinner>
 </div>
+
+
+<km-overlay-terminal *ngIf="showTerminal"
+                     (close)="toggleTerminal()">
```

Comment:
please add id to this element, so it would be easy for testing.

Reply:
There is no unit testing done! so therefore it is not added.

Reply:
I agree with @KhizerRehan here. Let's not add things that we don't use. If the test will be written in the future it is really simple to add new ID. Otherwise, we will end up with a lot of redundant things in the code.